library/src/morpheus-master/js/src/sakai.morpheus.connection-manager.js

```diff
@@ -1,6 +1,6 @@
 (function ($) {
 
-    if (!portal.loggedIn){
+    if (!portal.loggedIn || !(portal.bullhorns && portal.bullhorns.enabled)){
         return;
```

Comment:
Wouldn't this disable the connection manager if bullhorns are disabled?

Reply:
Hi, @adrianfish, as long as I see on this sakai property description https://github.com/sakaiproject/sakai/blob/master/config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties#L5063 bullhorns and connection manager are enabled or disabled with the same property "Add academic and social alerts icons" so yes, that would disable connection manager because it is controlled by same portal (sakai) property.

Even with that property disabled, i18n AJAX calls are being made. Maybe this should be loaded by bullhorns / connection manager but, right now, it is on morpheus so it is running everywhere twice for every page.

Reply:
Sorry, **absolutely my mistake**. Meroño and I mixed up connection bullhorn with that connection manager that it is on profile popup. I'm going to change it to make sure connection manager is not broken.

Can you tag this PR as WIP ? Tomorrow monday is a holiday here, I'll upload changes on tuesday morning.

Reply:
I'll save this i18n data on SessionStorage too so it wouldn't be neccessary to make the AJAX call on every page.

Thanks, @adrianfish 👍

Reply:
@SedueRey No problem whatsoever, that's why we do code review! The session storage bit is a great modification, nice work. I've set WIP on the PR.